lbz/api_base.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any
+
+
+class APIBase:
+    def react(self) -> Any:
```

Comment:
Also, during the discussion, we can think of how to make it more obvious that the handle will be triggered by `react()` 😉